includes/win8-appname-long-md.md

```diff
@@ -1 +1 @@
-Windows 8.x Store
\ No newline at end of file
+UWP
```

Comment:
I wouldn't change the content of this include file since it's probably used in other places to refer to Windows 8.x store. Instead, just change right on the text where the include file is used to say UWP.

Reply:
@mairaw I've removed the include file from the PR. We're actually now removing all mention of UWP/WIndows Store apps from this set of docs. (I don't understand why this comment that was made 7 days ago only yesterday appeared for me...!)